waterloop-site/src/theme/global.scss

```diff
@@ -710,7 +714,8 @@ h1 {
   align-items: center;
   justify-content: center;
   flex-wrap: wrap;
-  margin: 90px auto;
+  margin: auto auto;
+  margin-bottom: 1vh;
```

Comment:
I'd recommend changing this to "10vw" and removing "margin: auto auto"

Reply:
![image](https://user-images.githubusercontent.com/42826571/88230579-a5cefb00-cc40-11ea-91ae-cf91a9ef8fdf.png)

I think 10vw results in too much space, we could go for a smaller value probably.

Reply:
A smaller value works too :)